### PR TITLE
Fixes for doctor, unused parameter cleanup, help cleanup.

### PR DESCRIPTION
**Pull request by nathanagood**
- Removed file
- Update README.md
- Added help for required parameters, fixed doctor bug, and fixed name filter
- Fixed owner filter and issue with quotes not coming through for parameters.
- Minor help and parameter cleanup and two new CLI tests.

### CODE REVIEW

File: cmd/list.go [hunk 1]
```diff
@@ -83,12 +83,16 @@ deployed to a cluster.
 Example:
 
     itz list pipelines
+	itz list pipleines -o user:example.owner@ibm.com
```
Comment (davidstacy) [inline cmd/list.go:86]: spelling?

File: cmd/list.go [hunk 2]
```diff
@@ -83,12 +83,16 @@ deployed to a cluster.
...
+	itz list pipleines -n "Deployer CP4S 1.10"
```
Comment (davidstacy) [inline cmd/list.go:87]: possible typo

File: cmd/list.go [hunk 3]
```diff
@@ -14,7 +14,7 @@ import (
 
 var createdOnly bool
 var listAll bool
-var componentName string
+var componentNames []string
```
Comment (davidstacy) [inline cmd/list.go:17]: should there be a space between componentNames and []string?

**Review by davidstacy — CHANGES_REQUESTED** [review]
couple of things to check.

File: cmd/list.go (hunk 1 above)
Comment (nathanagood) [reply cmd/list.go]: Nice catch! Updated

File: cmd/list.go (hunk 2 above)
Comment (nathanagood) [reply cmd/list.go]: Updates this one too

File: cmd/list.go (hunk 3 above)
Comment (nathanagood) [reply cmd/list.go:17]: I changed from a single value to multiple possible; this is valid go syntax for the array.

**Review by davidstacy — APPROVED** [review]
LGTM

**Pull request merged**